crates/pallet-erc20-support/Cargo.toml

```diff
@@ -9,12 +9,21 @@ codec = { workspace = true, package = "parity-scale-codec", features = ["derive"
 frame-support = { workspace = true }
 frame-system = { workspace = true }
 scale-info = { workspace = true, features = ["derive"] }
+sp-std = { workspace = true }
```

Comment:
```suggestion
```

This should not be required since we can reach it via `frame_support::sp_std`.

Reply:
https://github.com/humanode-network/humanode/pull/809/commits/9673046a9a74da360b77cbb0b3aed8fd438ae40e